doc/howtos/rdtraining/E_unittest.rst

```diff
@@ -0,0 +1,212 @@
+.. _howto/rdtraining/E_unittest:
+
+=============================
+Advanced E: Python Unit Tests
+=============================
+
+.. tip:: This tutorial assumes you followed the Core Training.
+
+  To do the exercise, fetch the branch 14.0-core from the repository XXX.
+  It contains a basic module we will use as a starting point
+
+
+**Reference**:
+`Odoo's Test Framework: Learn Best Practices <https://www.youtube.com/watch?v=JEIscps0OOQ>`__
+(Odoo Experience 2020) on Youtube.
+
+Writing tests is a necessity for multiple reasons. Here is a non exhaustive list:
+
+* Ensure it will not be broken in the future
+* Define the scope of your code
+* Give examples of use cases
+* It is one way to technically document the code
+* Help you develop by defining your goal before working towards it
+
```

Comment:
I think before mentioning integration bots, it could be interesting to explain how a test can be run locally.